ix-dev/community/windmill/templates/test_values/basic-values.yaml

```diff
@@ -0,0 +1,70 @@
+windmill:
+  postgres_image_selector: postgres_16_image
+  web_host: localhost
+  db_password: changeme
+  base_url: ":80"
+  address: ""
+  log_max_size: "20m"
+  log_max_file: "10"
+  additional_envs: []
+
+run_as:
+  user: 568
+  group: 568
+
+network:
+  web_port: 80
+  smtp_port: 25
```

Comment:
You are using an old style port config.
Please see apps like umami on the new style